test/conformance/testdata/conformance.yaml

```diff
@@ -1644,6 +1644,15 @@
     the NodePort.
   release: v1.16
   file: test/e2e/network/service.go
+- testname: Service, preserve UDP traffic when pod backing a service cycles
```

Comment:
There is a process to promote tests to conformance 
https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/conformance-tests.md#conformance-testing-in-kubernetes

> > The process to add new conformance tests is intended to decouple the development of useful tests from their promotion to conformance:
> 
> Contributors write and submit e2e tests, to be approved by owning SIGs
> Tests are proven to meet the conformance test requirements by review and by accumulation of data on flakiness and reliability
> A follow up PR is submitted to promote the test to conformance